terminus-ui/drawer/src/drawer-header.component.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  ViewEncapsulation,
+} from '@angular/core';
+
+/**
+ * The drawer header UI Component
+ *
+ * @example
+ * <ts-drawer-header>
+ *   THIS IS MY HEADER
```

Comment:
Example docs indentation needs to be 13 spaces because of how TypeScript handles indents in docs.